SPECS/systemd/systemd.spec

```
@@ -39,9 +39,8 @@ Requires:       libgcrypt
 Requires:       lz4
 Requires:       pam
 Requires:       xz
-
+Requires(pre):  audit-libs
```

Re-adding suggestions from our e-mail discussion:

```suggestion
Requires(post):  audit-libs
Requires(post):  pam
Requires(post):  util-linux
```

I can't remove the `Requires: pam` above in these suggestions, sorry.:)

---

Why wouldn't Requires(pre) be correct?  Isn't the hint saying that it must come before installing and helps break the cycle?  Post seems to put us back in the situation we had?